### PR TITLE
user-stats

### PR DESCRIPTION
**Pull request by darmhoo**

### CODE REVIEW

File: src/views/admin/LeaderBoard.vue [hunk 1]
```diff
@@ -246,18 +313,73 @@
...
+    await this.getAgents();
+    this.sortByPurchased();
+    this.totalRegistered = this.getTotalRegistered();
+
+  },
   methods: {
     generateRandomColor() {
       return "#" + Math.floor(Math.random() * 16777215).toString(16);
     },
+    sortByPurchased(){
```
Comment (teebarg) [inline src/views/admin/LeaderBoard.vue:344]: you don't need this

File: src/views/admin/LeaderBoard.vue [hunk 2]
```diff
@@ -246,18 +313,73 @@
...
+      totalRegistered: 0,
+      option: ''
+    };
   },
   components: {
     Sidebar,
   },
+  async created() {
+    await this.getAgents();
+    this.sortByPurchased();
```
Comment (teebarg) [inline src/views/admin/LeaderBoard.vue:336]: set this.option to purchased  and call changeOrder method instead

Comment (darmhoo) [reply src/views/admin/LeaderBoard.vue:336]: ok

File: src/views/admin/LeaderBoard.vue [hunk 3]
```diff
@@ -246,18 +313,66 @@
...
+      totalRegistered: 0,
+      option: ''
+    };
   },
   components: {
     Sidebar,
   },
+  async created() {
+    await this.getAgents();
+    this.option = 'purchased';
```
Comment (teebarg) [inline src/views/admin/LeaderBoard.vue:336]: set it inside data

**Review by teebarg — APPROVED** [review]

**Pull request merged**